config/tcs_config.toml

```diff
@@ -30,80 +30,15 @@ LogFile  = "__screen__"
 # Set this flag to 1 for sync workorder execution.Else set it to 0
```

Comment:
Who is going to use this tcs_config.toml file?

Reply:
Other components than than enclave manager along with a few scripts.